geode-for-redis/src/main/java/org/apache/geode/redis/internal/GeodeRedisServer.java

```diff
@@ -69,13 +69,13 @@
   /**
    * Constructor for {@code GeodeRedisServer} that will configure the server to bind to the given
    * address and port.
```

Comment:
This comment should not reference the address and port, but rather the `RedisConfiguration`.

Reply:
Updated.